.pre-commit-config.yaml

```diff
@@ -32,10 +34,11 @@ repos:
     rev: v2.7.0
     hooks:
       - id: validate_manifest
-  - repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v1.5.4
-    hooks:
-      - id: autopep8
+  # - repo: https://github.com/pre-commit/mirrors-autopep8
```

Comment:
Should we clean this up ?

Reply:
We were getting some errors with the generated code.  We can look at it in dev.